catapult-sdk/src/modelBinary/embeddedEntityCodec.js

```diff
@@ -31,8 +31,11 @@ const embeddedEntityCodec = {
 	 */
 	deserialize: parser => {
 		const entity = {};
+		entity.embeddedTransactionHeader_Reserved1 = parser.uint32();
```

Comment:
This is missing `size: uint32`. EmbeddedTransaction schema inlines `SizePrefixedEntity` to header: https://github.com/nemtech/catbuffer/blob/fushicho2/schemas/transaction.cats#L17

Reply:
``blockHeaderCoded ``and ``transactionsCodec`` does not treat either ``size``.  Perhaps they are filtering it before, but definitely something to double-check before merging.

Reply:
it's not missing the size